packages/kbn-config/src/deprecation/deprecation_factory.ts

```diff
@@ -56,6 +56,26 @@ const _rename = (
   return config;
 };
 
+const _copy = (
```

Comment:
I have the feeling this package is missing some CODEOWNERS entry. Feels most likely like @elastic/kibana-platform should own this? Maybe would be nice to get a quick review here nevertheless.

Reply:
We did it! see here https://github.com/elastic/kibana/pull/79627#discussion_r512649107

Reply:
Sorry for the noise :) (Maybe someone from platform team still wants to add this package to their codeowners)